src/main/java/Duke.java

```diff
@@ -1,10 +1,99 @@
+import java.util.List;
+import java.util.Scanner;
+import java.util.*;
```

Comment:
Maybe it is better to be explicit on which class we want when importing instead of import java.util.*?